### ISSUE TITLE
SEPATransfer Call to a member function format() on null

### ISSUE THREAD

**Issue by timrasche** [nemiah/phpSepaXml#6]
Hello there!

on cross-europe transfers the bank returned that the exection date MUST be empty.

**Comment by nemiah** [comment]
Yes, it seemed correct but it is not. Try the new release please.

**Comment by timrasche** [comment]
Thanks but, 1999-01-01 is also an value and not empty/undefined. Further in SEPATransfer.php file the addCreditor function does not work in this szenario. format() on null variable..

**Comment by seem2810** [comment]
It should be more dynamic... 
Because your revert my changes brokes: 
https://github.com/nemiah/phpFinTS/issues/445#issuecomment-2273952444 

So it should be checked if the receiver in europe and a execution date is set, then set the date.
if in europe, no date set, set to 1991
if not in europe - set to null 

Please change it.

**Comment by timrasche** [comment]
Any news on this?
